config/application-copy.yml

```diff
@@ -0,0 +1,9 @@
+CLIENT_ID: ''
+CLIENT_SECRET: ''
+REDIRECT_URI: ''
+OAUTH_URL: ''
+BASE_URL: ''
+
+#Copy these keys into your followed application.yml file. The application.yml file is created after the installation of the Figaro gem (see Readme for more instructions)
```

Comment:
There should be a space at the start of each of these comment lines, after the `#`